starlite/testing.py

```diff
@@ -288,6 +288,7 @@ def __init__(
         server: str = "test.org",
         port: int = 3000,
         root_path: str = "",
+        scheme: str = "http",
```

Comment:
I missed `scheme` in the previous PR. I realized it because `create_test_request` uses it :)